graphql/endorsement.graphql

```diff
@@ -0,0 +1,39 @@
+extend type Query {
+    specialEndorsements: [SpecialEndorsement!]! @all
+    specialEndorsement(id: ID @eq): SpecialEndorsement @find
+    specialEndorsementsByUser(user_id: ID @eq): Assignment @find
```

Comment:
this should return a list of SpecialEndorsement, not a single assignment pivot

Reply:
Now been addressed :)